test/httptests.ts

```diff
@@ -14,7 +14,7 @@ describe('Http Tests', function () {
     let _httpbin: httpm.HttpClient;
 
     before(() => {
-        _http = new httpm.HttpClient('typed-test-client-tests');
+        _http = new httpm.HttpClient('typed-rest-client-tests'/*, null, {proxy: { proxyUrl: "http://127.0.0.1:8888" } }*/);
```

Comment:
commented out code?

Reply:
Yeah it's just for when we want to use Fiddler to trace things. Left it commented so we can uncomment just for that testing case. Happy to remove if it's confusing.

Reply:
the other option is to formalize it - make the tests respect proxy envvar

Reply:
Do we have a normal process that we follow for setting them? I can see respecting the env var for the lib itself but for tests it makes sense to me to be explicit/easily configurable. Otherwise they could pass on my machine and fail on yours if the env var is set.

Reply:
Adding offline discussion notes: we will add a custom environment variable to manage this.